prs/WeatherData.test.cs

```diff
@@ -0,0 +1,9 @@
+namespace Sample {
+    [TestClass]
+    public class WeatherDataTest {
+        [TestMethod]
+        public static void Test1 {
+            // TODO: write tests 😝
```

Comment:
Can we write some unit tests for this change? Looks like this might be leftovers or copy/pasta